canonical-web-and-design/maas-ui
# Expose websocket errors

Done:
- Consolidate App and Main.
- Expose websocket errors. Fixes: #39.

QA:
- Do something that causes a websocket error e.g. stop your maas and reload the UI.
- You should see an error.
- Check that still see loading messages and logged out messages.
